src/Stripe.net/Entities/Accounts/AccountCapabilities.cs

```diff
@@ -12,6 +12,13 @@ public class AccountCapabilities : StripeEntity<AccountCapabilities>
         [JsonProperty("au_becs_debit_payments")]
         public string AuBecsDebitPayments { get; set; }
 
+        /// <summary>
+        /// The status of the Bacs Direct Debits payments capability of the account, or whether the
```

Comment:
wonder if we want to add the values in the comment here? `active`, `inactive`, or `pending`? non blocking.

Reply:
I guess that's the same for all of these so maybe doesn't make sense to add to all.

Reply:
Yeah it'd be repeated and the alignment of the comments is often different based on the previous sentence so it's a pain to do by hand :p